src/main/java/se/sundsvall/supportmanagement/integration/db/model/CommunicationEntity.java

```diff
@@ -0,0 +1,258 @@
+package se.sundsvall.supportmanagement.integration.db.model;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.Objects;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Id;
+import jakarta.persistence.Index;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import jakarta.persistence.UniqueConstraint;
+
+import se.sundsvall.supportmanagement.integration.db.model.enums.CommunicationType;
+import se.sundsvall.supportmanagement.integration.db.model.enums.Direction;
+
+
+@Entity
+@Table(name = "communication",
+	indexes = {@Index(name = "idx_errand_number", columnList = "errand_number")},
+	uniqueConstraints = {@UniqueConstraint(name = "uq_errand_number", columnNames = {"errand_number"})})
+public class CommunicationEntity {
+
+	@Id
+	@Column(name = "communication_id")
+	private String communicationID;
+
+	@Column(name = "errand_number")
+	private String errandNumber;
+
+	@Column(name = "direction")
+	@Enumerated(EnumType.STRING)
+	private Direction direction;
+
+	@Column(name = "external_case_id")
+	private String externalCaseID;
+
+	@Column(name = "subject")
+	private String subject;
+
+	@Column(name = "message_body")
+	private String messageBody;
+
+	@Column(name = "sent")
+	private OffsetDateTime sent;
+
+	@Column(name = "message_type")
+	@Enumerated(EnumType.STRING)
+	private CommunicationType communicationType;
+
+	@Column(name = "mobile_number")
+	private String mobileNumber;
```

Comment:
Is it possible to merge mobileNumber and email? Maybe introduce a "target"-column instead?

I.e. 
If communicationType is SMS, then target is a mobileNumber. 
If communicationType is EMAIL then target is an email-address

Otherwise one of these columns will always be empty.